SwagLyricsBot/swaglyrics_bot.py

```diff
@@ -29,18 +29,19 @@ async def on_ready():
     await bot.change_presence(activity=discord.Activity(name="you type $sl", type=3), status=discord.Status.online)
 
 
-def run():
+async def run():
     """
     Bot setup
     """
-    token = env_file.get()
-    bot.add_cog(DevCommands(bot))
-    bot.add_cog(GeneralCommands(bot))
-    bot.add_cog(LinksCommands(bot))
-    if 'DBL_TOKEN' in token:
-        bot.add_cog(TopGG(bot, token['DBL_TOKEN']))
-    if 'WEBHOOK_URL' in token:
-        logs.webhook_url = token['WEBHOOK_URL']
-    if 'WEBHOOK_ERROR_SUPERVISOR_ID' in token:
-        logs.error_supervisor = token['WEBHOOK_ERROR_SUPERVISOR_ID']
-    bot.run(token["BOT_TOKEN"])
+    async with aiohttp.ClientSession() as session:
```

Comment:
It wouldn't matter if you just use `session = aiohttp.ClientSession()` in this context since it will get closed automatically on exit. So it's probably cleaner to do that instead.

Reply:
what do you mean?

Reply:
Instead of
```
async with aiohttp.ClientSession() as session:
```
just make it:
```
session = aiohttp.ClientSession()
```

Reply:
makes sense, but I'd rather stick with the context manager to establish good practices instead of wondering if I forgot to close it or nah